### PR TITLE
ffmpeg: add version 5.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **ffmpeg/5.0**

retry.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/ffmpeg/all/conanfile.py [hunk 1]
```diff
@@ -630,6 +630,8 @@ def package_info(self):
                 self.cpp_info.components["avfilter"].frameworks.append("AppKit")
             if self.options.get_safe("with_coreimage"):
                 self.cpp_info.components["avfilter"].frameworks.append("CoreImage")
+            if tools.Version(self.version) >= "5.0" and tools.is_apple_os(self.settings):
```
Comment (SSE4) [inline recipes/ffmpeg/all/conanfile.py:633]:
```suggestion
            if tools.Version(self.version) >= "5.0" and tools.is_apple_os(self.settings.os):
```

**Comment by SSE4** [comment]
see https://github.com/conan-io/conan-center-index/pull/9467

**Comment by SpaceIm** [comment]
https://github.com/conan-io/conan-center-index/pull/9467 has been merged, it may fix issue in this PR.

**Review by SSE4 — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Comment by SpaceIm** [comment]
```
  Checking Build System
  Building Custom Rule C:/J/w/prod/BuildSingleReference/conan-center-index/recipes/ffmpeg/all/test_package/CMakeLists.txt
  test_package.c
  mp3lame.lib(lame.obj) : MSIL .netmodule or module compiled with /GL found; restarting link with /LTCG; add /LTCG to the link command line to improve linker performance
     Creating library C:/J/w/prod/BuildSingleReference/conan-center-index/recipes/ffmpeg/all/test_package/build/d9beaafb75177aced97f5ddd59a6fa859285033a/lib/test_package.lib and object C:/J/w/prod/BuildSingleReference/conan-center-index/recipes/ffmpeg/all/test_package/build/d9beaafb75177aced97f5ddd59a6fa859285033a/lib/test_package.exp
  Generating code
  Finished generating code
  test_package.vcxproj -> C:\J\w\prod\BuildSingleReference\conan-center-index\recipes\ffmpeg\all\test_package\build\d9beaafb75177aced97f5ddd59a6fa859285033a\bin\test_package.exe
  Building Custom Rule C:/J/w/prod/BuildSingleReference/conan-center-index/recipes/ffmpeg/all/test_package/CMakeLists.txt
ffmpeg/4.3.2 (test package): Running test()
```

There are still inconsistency in Visual Studio runtime, either in ffmpeg itself or in few dependencies. And libmp3lame is compiled with LTO.

**Comment by SpaceIm** [comment]
https://github.com/conan-io/conan-center-index/pull/9740 & https://github.com/conan-io/conan-center-index/pull/9741 should fix these inconsistencies

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**